test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -281,22 +281,49 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			step( 'Can see the Earn blocks', async function () {
 				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
 				await gEditorComponent.openBlockInserterAndSearch( 'earn' );
+				const shownItems = await gEditorComponent.getShownBlockInserterItems();
+
 				assert.strictEqual(
-					await gEditorComponent.isBlockCategoryPresent( 'Earn' ),
-					true,
-					'Earn (Jetpack) blocks are not present'
+					[
+						'Donations',
+						'OpenTable',
+						'Payments',
+						'Pay with PayPal',
+						'Premium Content',
+						'Pricing Table',
+					].every( ( v ) => shownItems.includes( v ) ),
+					true
 				);
+
 				await gEditorComponent.closeBlockInserter();
 			} );
 
 			step( 'Can see the Grow blocks', async function () {
 				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
 				await gEditorComponent.openBlockInserterAndSearch( 'grow' );
+				const shownItems = await gEditorComponent.getShownBlockInserterItems();
+
 				assert.strictEqual(
-					await gEditorComponent.isBlockCategoryPresent( 'Grow' ),
-					true,
-					'Grow (Jetpack) blocks are not present'
+					[
+						'Business Hours',
+						'Calendly',
+						'Form',
+						'Contact Info',
+						'Mailchimp',
+						'Revue',
+						'Subscription Form',
+						'Click to Tweet',
+						'Logos',
+						'Contact Form',
+						'RSVP Form',
+						'Registration Form',
+						'Appointment Form',
+						'Feedback Form',
+						'WhatsApp Button',
+					].every( ( v ) => shownItems.includes( v ) ),
```

Comment:
Ditto! Can we make the assertion per block here as well?

Reply:
👍🏻